spy/ast_dump.py

```diff
@@ -1,23 +1,29 @@
-from typing import Any, Optional
+from typing import TYPE_CHECKING, Any, Optional
 import ast as py_ast
+
+from spy.analyze.symtable import Color
 import spy.ast
 from spy.analyze.symtable import Symbol
 from spy.textbuilder import TextBuilder
+if TYPE_CHECKING:
+    from spy.vm.vm import SPyVM
 
 def dump(node: Any,
          *,
          use_colors: bool = True,
          fields_to_ignore: Any = (),
          hl: Any = None,
+         vm: 'SPyVM | None' = None,
```

Comment:
nitpick: so far I always used `Optional[T]` instead of `T | None`.
It's a very minor thing but I think there is value in being consistent.
(I _might_ be convinced that it's better to use `| None` instead of `Optional`, but then it's a topic for a different PR).